platform/server/src/schema/schema.graphql

```diff
@@ -64,6 +64,60 @@ type Mutation {
   deleteVolunteerAction(input: VolunteerActionInput): VolunteerAction!
   createRecipient(input: RecipientInput): Recipient!
   updateRecipient(input: RecipientInput): Recipient!
+  createProduct(input: ProductInput): Product!
+  updateProduct(input: ProductInput): Product!
+}
+
+"""
+Represents a join model between a recipient and product
+@model
+@crud.update: false
```

Comment:
We have bug to solve this in one annotation! THX

Reply:
Yes. I totally agree, and thought and tried 
```
@crud: false
```

but did not work. Any link to the existing bug?